yjit/src/stats.rs

```diff
@@ -382,7 +382,7 @@ fn rb_yjit_gen_stats_dict() -> VALUE {
         hash_aset_usize!(hash, "freed_code_size", freed_page_count * cb.page_size());
 
         // Compiled pages
-        hash_aset_usize!(hash, "compiled_page_count", cb.num_pages() - freed_page_count);
+        hash_aset_usize!(hash, "compiled_page_count", cb.num_mapped_pages() - freed_page_count);
```

Comment:
Maybe we should call this stat `live_page_count` to more directly represent its meaning.

Based on the name, I would assume that `compiled_page_count` is the number of pages that have been filled with code in total 🤔

Reply:
:+1: Renamed it https://github.com/ruby/ruby/pull/6644/commits/1489c03539fef086a4590cf8d8422c1b4e0fa755